cmake/FindQuickLintJsEmacs.cmake

```
@@ -0,0 +1,81 @@
+# Copyright (C) 2020  Matthew "strager" Glazar
+# See end of file for extended copyright information.
+
+if (QUICK_LINT_JS_EMACS)
+  set(QUICK_LINT_JS_EMACS "${QUICK_LINT_JS_EMACS}")
+else ()
+  find_program(QUICK_LINT_JS_EMACS "emacs")
+endif ()
```

`find_program` already checks if the program was found. No need to check yourself:
```suggestion
find_program(QUICK_LINT_JS_EMACS "emacs")
```